LDKNodeMonday/View/Home/OnboardingView.swift

```diff
@@ -23,130 +27,78 @@ struct OnboardingView: View {
-                    if viewModel.seedPhraseArray != [] {
-                        SeedPhraseView(
-                            words: viewModel.seedPhraseArray,
-                            preferredWordsPerRow: 2,
-                            usePaging: true,
-                            wordsPerPage: 6
-                        )
-                    }
+                    Image(systemName: "bolt.horizontal.fill")
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .foregroundColor(.accent)
+                        .frame(width: 150, height: 150, alignment: .center)
+                        .padding(40)
+                    Text("Monday Wallet")
+                        .font(.largeTitle.weight(.semibold))
+                    Text("An example bitcoin wallet\npowered by LDK Node")
+                        .font(.body)
+                        .multilineTextAlignment(.center)
+                        .fixedSize(horizontal: false, vertical: true)
                 }
-                .padding()
 
                 Spacer()
 
-                Button {
+                // Buttons for creating and importing wallet
+
+                Button("Create wallet") {
                     viewModel.saveSeed()
                     isFirstTime = false
-                } label: {
-                    Text("Start Node")
-                        .bold()
-                        .foregroundColor(Color(uiColor: UIColor.systemBackground))
-                        .frame(maxWidth: .infinity)
-                        .minimumScaleFactor(0.9)
-                        .padding(.all, 8)
                 }
-                .frame(width: 200, height: 25)
-                .buttonStyle(BitcoinFilled(tintColor: viewModel.buttonColor, isCapsule: true))
-                .disabled(!isFirstTime && viewModel.seedPhrase.isEmpty)
-                .padding(.all, 25)
-
-            }
+                .buttonStyle(
+                    BitcoinFilled(
+                        tintColor: .accent,
+                        isCapsule: true
+                    )
+                )
 
-        }
-        .alert(isPresented: $showingOnboardingViewErrorAlert) {
-            Alert(
-                title: Text(viewModel.onboardingViewError?.title ?? "Unknown"),
-                message: Text(viewModel.onboardingViewError?.detail ?? ""),
-                dismissButton: .default(Text("OK")) {
-                    viewModel.onboardingViewError = nil
+                Button("Import wallet") {
+                    showingImportWalletSheet.toggle()
+                }
+                .buttonStyle(BitcoinPlain(tintColor: .accent))
+                .sheet(isPresented: $showingImportWalletSheet) {
+                    ImportWalletView(viewModel: viewModel)
                 }
-            )
-        }
+
+            }.dynamicTypeSize(...DynamicTypeSize.accessibility2)  // Sets max dynamic size for all Text
```

Comment:
I like this, remind me to do this if you see me not doing it in the future on my PR's